ng2-components/ng2-alfresco-datatable/src/components/datatable/datatable.component.ts

```diff
@@ -236,7 +236,8 @@ export class DataTableComponent implements AfterContentInit, OnChanges {
         let event = new DataCellEvent(row, col, []);
         this.showRowActionsMenu.emit(event);
 
-        return this.checkPermissions(row, event.value.actions);
+        return event.value.actions;
+        // return this.checkPermissions(row, event.value.actions);
```

Comment:
So at this point, I have two questions:
- this thing removes a functionality and there are no failing test? so are missing...
- this PR introduce another bug in order to solve another? Maybe is better fix the original problem instead to comment out the functionality?